Lib/subprocess.py

```diff
@@ -978,6 +978,21 @@ def __init__(self, args, bufsize=-1, executable=None,
 
             raise
 
+    def __repr__(self):
+        max_args_length = 30
+        args = ' '.join(self.args)
+
+        if len(args) > max_args_length:
+            args = ('%s...' % args[:max_args_length]).strip()
```

Comment:
This is regarding all formatted strings: Since we're in Python 3.9, why not use f-strings?